cime_config/machines/config_machines.xml

```diff
@@ -3133,6 +3133,114 @@
     </environment_variables>
   </machine>
 
+  <machine MACH="chicoma-cpu">
+    <DESC>Chicoma CPU-only nodes at LANL IC. Each node has 2 AMD EPYC 7H12 64-Core (Milan) 512GB</DESC>
+    <NODENAME_REGEX>ch-fe.*.lanl.gov</NODENAME_REGEX>
```

Comment:
I needed to change this to 
ch-fe*
since the fully qualified hostname the system returned was ch-fe#.LOCAL for some reason